src/core/swap/swap-api.js

```diff
@@ -123,16 +129,30 @@ export async function fetchSwapQuote(
  * Picks the best quote out of the available choices.
  * Exported so we can unit-test it.
  */
+
 export function pickBestQuote(
   quotes: EdgeSwapQuote[],
   preferPluginId: string | void,
-  promoCodes: EdgePluginMap<string>
+  promoCodes: EdgePluginMap<string>,
+  preferType?: EdgeSwapPluginType
```

Comment:
Extra whitespace? Also, can we just pass the whole `EdgeSwapRequestOptions` object in, since we pretty much need all of it?

Reply:
Flow isn't working on my VScode 🥲. I'm having to manually format then hope `yarn flow` `yarn lint` doesnt catch anything. Can't wait for that TS conversion! 👍 

Sure. I didn't want to have to redo all unit tests causing extra diffs but this makes sense.

Reply:
You can also do `yarn fix` on the CLI to auto-format everything.